src/main/java/com/SAS/User/Coach.java

```diff
@@ -42,7 +42,7 @@ public User getUser() {
      *
      * @return level - int
      */
-    public Integer getLevel() {
+    public int getLevel() {
         return level;
     }
 
```

Comment:
why did you change Integer to int?

Reply:
Because it shouldn't be an Integer, there is not reason.

Reply:
the Integer class is provided so that values can be boxed/unboxed in a pure OO manner. use int where appropriate unless you specifically need to use it in an OO way; in which case Integer is appropriate

However, very different things are going on under the covers here. An int is a number
 an > Integer is a pointer that can reference an object that contains a number

An int is not an object and cannot passed to any method that requires objects. A common case is in using the provided collection classes ( List , Map , Set ) - though it is possible to write versions of these classes that provide similar capabilities to the object versions. The wrapper classes ( Integer , Double , etc) are frequently required whenever introspection is used (such as in the reflection API)

concider that and update me on your decission :)

Reply:
I think I'll leave like this, since it not a collection.

Reply:
i only gave you one example, but as you wish.